vendor/psr/http-message/composer.json

```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": "^7.2 || ^8.0"
```

Comment:
In the release 2.0, we will be compatible with php 7.1+ (not php 7.2)

Reply:
PR is updated